arangod/Pregel/Status/Status.h

```diff
@@ -55,17 +74,30 @@ auto inspect(Inspector& f, Status& x) {
           .transformWith(inspection::TimeStampTransformer{}),
       f.field("verticesLoaded", x.verticesLoaded),
       f.field("edgesLoaded", x.edgesLoaded),
-      f.field("memoryUsed", x.memoryBytesUsed));
+      f.field("memoryUsed", x.memoryBytesUsed),
+      f.field("verticesStored", x.verticesStored));
 }
 
 struct Observables {
-  std::atomic<std::size_t> verticesLoaded{0};
-  std::atomic<std::size_t> edgesLoaded{0};
-  std::atomic<std::size_t> memoryBytesUsed{0};
+  std::optional<std::atomic<std::size_t>> verticesLoaded;
+  std::optional<std::atomic<std::size_t>> edgesLoaded;
+  std::optional<std::atomic<std::size_t>> memoryBytesUsed;
+  std::optional<std::atomic<std::size_t>> verticesStored;
```

Comment:
Is this correct? Imagine one thread observing that `verticesLoaded` is `std::nullopt` and deciding to emplace `std::atomic<std::size_t>{5}` in there; another thread also observes `std::nullopt` and puts `std::atomic<std::size_t>{15}` into `verticesLoaded`, when you actually want `20` as the total you will have either `5` or `15` depending on who is faster.